grid-demo/src/app/angular/helpers/datagrid-private-member.helpers.ts

```diff
@@ -0,0 +1,23 @@
+import { ElementRef } from '@angular/core';
+import { ClrDatagrid } from '@clr/angular';
+
+interface ClrDatagridPrivateMembers {
```

Comment:
This comment is general for the file. I see this adds some type safety, but we are still accessing private members.
What's the idea of this refactor? Do we gain anything long term?

Reply:
I just moved the `datagrid as any` casts to a single place and adds type safety. Hopefully, this all gets removed in the future, but I wanted to add some safety in the meantime.